core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2111,14 +2126,17 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    IllegalArgumentException siteNotFound(String siteName);
 
    @LogMessage(level = WARN)
-   @Message(value = "Cleanup failed for Cross-Site state transfer. In case of issues, invoke operation cancel-push-state(%s).", id = 618)
+   @Message(value = "Cleanup failed for cross-site state transfer. In case of issues, invoke operation cancel-push-state(%s).", id = 618)
+   @Description("It was not possible to clean...")
```

Comment:
@pruivo Could you please give a bit of extra detail for this message as well as the one after it? Seems like they are both more or less the same thing but for different sites, either local or remote.

Does the cleanup operation take place after cross-site state transfer is finished? What does it involve?

When we say "in case of issues" what are the issues that are likely to occur? How could the user identify if there were issues?

Reply:
One node (can be any node!) coordinates the cross-site state transfer and, when the state transfer completes (or fails), it sends these cancel commands to the local and remote sites to clean up any pending state. 

Since the network is involved, there is a possibility of timeout exceptions or other network exceptions. If the state is incorrect (i.e. nodes are still reporting that are receiving or sending state), the user can invoke the cancel again to clean up anything pending.